api/constants/constants.go

```diff
@@ -146,6 +146,10 @@ const (
 	// DatabaseCAMinVersion is the minimum Teleport version that supports Database Certificate Authority.
 	DatabaseCAMinVersion = "10.0.0"
 
+	// OpenSSHCAMinVersion is the minimum Teleport version that supports OpenSSH Certificate Authority.
+	// TODO: change this to be correct before merging
```

Comment:
It looks like it's been decided that this goes in 12 so this can be removed.

Reply:
Yep, thanks for the catch